code/source/playfab/PlayFabPluginManager.cpp

```diff
@@ -84,7 +84,7 @@ namespace PlayFab
 #ifdef PLAYFAB_PLATFORM_XBOX
         return std::make_shared<PlayFabIXHR2HttpPlugin>();
 #elif defined(PLAYFAB_PLATFORM_WINDOWS)
-        return std::make_shared<PlayFabWinHttpPlugin>();
+        return std::make_shared<PlayFabHCHttpPlugin>();
```

Comment:
I think this would be better if it was a new if-above this one, which was like:

```
#elif defined(PLAYFAB_PLATFORM_WINDOWS_GDK)
        return std::make_shared<PlayFabHCHttpPlugin>();
#elif defined(PLAYFAB_PLATFORM_WINDOWS)
        return std::make_shared<PlayFabWinHttpPlugin>();
```

Reply:
Switch, PlayStation, Linux, Mac, etc are all going to have different http layers. It's actually mandatory to use the platform specific http layer for all game consoles, you're not allowed to bring in your own, for better or worse.

Reply:
we already have libHttpClient implementations for those platforms.  They live in NDA'd repos currently so we will need to figure out exact logistics, but using libHttpClient for all platforms should work.

---
In reply to: [590887266](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590887266) [](ancestors = 590887266)

Reply:
As mentioned this whole plugin architecture is likely to change so "won't fix"ing for now

---
In reply to: [590886609](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590886609) [](ancestors = 590886609)